src/rhelocator/update_images/aws.py

```diff
@@ -114,3 +114,65 @@ def parse_image_name(image_name: str) -> dict[str, str]:
         return matches.groupdict()
 
     return {}
+
+
+def format_all_images() -> object:
+    """Retrieve all AWS images from all regions and return a simplified data
+    representation.
+
+    Returns:
+        JSON like structure containting a list of streamlined image
+        information on all regions.
+    """
+    formated_images: list[dict[str, str]] = []
```

Comment:
Not a huge deal, but this should probably be `formatted_images` (two t's)

Reply:
"The fault, dear Brutus, is not in our <disabled linter>, but in our <ability to write plain English>"